server_src/runners/state.js

```diff
@@ -20,7 +21,9 @@ const { JobClean } = require("../lib/dataFunctions/jobClean.js");
 const { FileClean } = require("../lib/dataFunctions/fileClean.js");
 const { FilamentClean } = require("../lib/dataFunctions/filamentClean.js");
 const { PrinterTicker } = require("./printerTicker.js");
-const {checkPluginManagerAPIDeprecation} = require("../utils/compatibility.utils");
+const {
+  checkPluginManagerAPIDeprecation
+} = require("../utils/compatibility.utils");
```

Comment:
Honestly probably not going to be utilised anywhere else but rather than making state.js any bigger XD